makeConfig.m

```diff
@@ -68,6 +68,9 @@
 config.minCallPrefix  = 'min';
 config.signCallPrefix = 'sign';
 
+% ignore string that lets the preprocessing ignore/skip an if-statement
+config.preprocess_ignorestring = 'ifdiff:ignore';
```

Comment:
sollte ifdiff::ignore sein (doppelter Doppelpunkt)
Hintergrund:
Verwendet man nicht im normalen Schreiben

The problem with ifdiff:
ignore a statement is complicated

Wird ohne Zeilenumbruch zu
The problem with ifdiff:ignore a statement is complicated

Und schon steht ein Spezialkommando da.

Alle ifdiff-Kommandos (bisher gibt's nur eines) sollten mit "ifdiff::" eingeleitet werden.